data/speakers.json

```diff
@@ -0,0 +1,80 @@
+{
+  "Mike Rogers": {
+    "profile": {
+      "github_username": "MikeRogers0"
```

Comment:
The cool thing about GitHub is we can pull a bunch more data about the speaker via their API without an API key.